libcxx/test/std/time/time.hash/time.hash_enabled.pass.cpp

```diff
@@ -0,0 +1,55 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17
```

Comment:
Recommended: 


```suggestion
// REQUIRES: std-at-least-c++26
```

Isn't this C++26 (same as bellow)?